edx_toggles/tests/test_waffle.py

```diff
@@ -29,6 +29,17 @@ def test_constructor(self):
         self.assertEqual("module1", waffle.module_name)
         self.assertEqual(1, len(NaiveWaffle.get_instances()))
 
+    def test_toggle_methods(self):
```

Comment:
Could we rename this to `test_toggle_methods_with_enabled_waffle` and add another method called `test_toggle_methods_with_disabled_waffle` that inverts all the assertions? 

In the new test method, I'm guessing you can just patch `is_enabled` to return False, like this example: https://github.com/openedx/edx-platform/blob/ddb092c07c6e2b663dc3d52e7197b42f7f017988/lms/djangoapps/experiments/testutils.py#L16.

Reply:
@robrap I just added the inverted toggle test case as requested by you.